### PR TITLE
DBS: Überblick

### PR DESCRIPTION
**Pull request by Steve2955**

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- DBS-DORENDORF/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: DBS-DORENDORF/README.md [hunk 1]
```diff
@@ -1,9 +1,83 @@
...
+  - [Phasen des Entwurfprozesses](#phasen-des-entwurfprozesses)
+  - [Ableitung des Datenbankschemas aus einer verbalen Spezifikation](#ableitung-des-datenbankschemas-aus-einer-verbalen-spezifikation)
+
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 <!--newpage-->
+
+# Überblick über den Datenbankentwurfsprozess
+
+Der Entwurf eines Datenbank aufwendiger Prozess, der mit großer Sorgfalt vorgenommen werden sollte, da Fehler später nur mit hohem Aufwand beseitigt werden können.
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:19]:
```suggestion
Der Entwurf einer Datenbank ist ein aufwendiger Prozess, der mit großer Sorgfalt vorgenommen werden sollte, da Fehler später nur mit hohem Aufwand beseitigt werden können.
```

File: DBS-DORENDORF/README.md [hunk 2]
```diff
@@ -1,9 +1,83 @@
...
+
+- **Anwenderorientiertheit:** Beteiligung der Anwender am Entwurfsprozess; Formulierung der Datenmodelle in der Begriffswelt der Anwender
+- **Determiniertheit:** genau ein Datenmodell für eine Problemstellung und einen Umweltausschnitt $\rightarrow$ unabhängig von Interpretation
+- **Teilbarkeit:** Gesamtschema muss in einzelne Teilschemata aufteilbar sein
+- **Integrierbarkeit:** Schemas, die nur einen Teil des Umweltausschnittes darstellen, müssen in das gesamte Schema integriert werden können
+
+## Entwicklung von Datenbankschemata
+
+- Berücksichtigung zweier Entwurfsphasen, bei der Entwicklung datenbankgestützter IT-Lösungen
+  - einmalliger Datenbankentwurfsprozess (DEP)
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:31]:
```suggestion
  - einmaliger Datenbankentwurfsprozess (DEP)
```

File: DBS-DORENDORF/README.md [hunk 3]
```diff
@@ -1,9 +1,83 @@
...
+  - n-maliger Softwareentwicklungsprozess (SEP) für Anwendungen, die auf die zentrale DB zugreifen
+- der DEP bildet mit Hilfe von Daten- und Beschreibungsmodellen ein Umweltausschnitt (Miniwelt) auf ein Schema ab
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:33]:
```suggestion
- der DEP bildet mit Hilfe von Daten- und Beschreibungsmodellen einen Umweltausschnitt (Miniwelt) auf ein Schema ab
```

File: DBS-DORENDORF/README.md [hunk 4]
```diff
@@ -1,9 +1,83 @@
...
+- jeder SEP hat meist eigene Sicht auf Datenbank $\rightarrow$ zusammenfassen zu einem konzeptuellen Informationschema
+- durch Transformationsregeln kann das konzeptuelle in ein datenbankspezifische Schema umgewandelt werden
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:35]:
```suggestion
- durch Transformationsregeln kann das konzeptuelle in ein datenbankspezifisches Schema umgewandelt werden
```

File: DBS-DORENDORF/README.md [hunk 5]
```diff
@@ -1,9 +1,83 @@
...
+- standardisierte, vom späteren Datenmodell unabhängige Darstellungsformen
+- z.B. *Entity-Relationship-Modell*
+
+**Ableitung eines dem Zieldatenmodell angepassten Informationsschemas**
+
+Ableiten einer Informationsstruktur $\rightarrow$ Abbildung von Objekten und Beziehungen zu Struktur die von einem DBMS verwaltet werden kann
+
+**Physischer Entwurf**
+
+- Anpassung des abgeleiteten Schemas an das konrekt zu verwendende DBMS und die dazugehörige Datenbasis
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:79]:
```suggestion
- Anpassung des abgeleiteten Schemas an das konkrekt zu verwendende DBMS und die dazugehörige Datenbasis
```

File: DBS-DORENDORF/README.md [hunk 6]
```diff
@@ -1,9 +1,83 @@
...
+- Zuordnung einzelner Attribute zu bestimmten Datentypen
+- Festlegung von Integritätsbedingungen und referenziellen Integritäten
+- Betrachtung von Forderungen nach Datenschutz und Datensicherheit (u.a. vergabe von Zugriffsrechten)
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:82]:
```suggestion
- Betrachtung von Forderungen nach Datenschutz und Datensicherheit (u.a. Vergabe von Zugriffsrechten)
```

File: DBS-DORENDORF/README.md [hunk 7]
```diff
@@ -1,9 +1,83 @@
...
+- Nach dieser Phase erfolt die implementierung der Datenbank auf einem konkreten Rechnersystem
```
Comment (RvNovae) [inline DBS-DORENDORF/README.md:83]:
```suggestion
- Nach dieser Phase erfolgt die Implementierung der Datenbank auf einem konkreten Rechnersystem
```

**Review by RvNovae — APPROVED** [review]
Typos, ansonsten supi :)

**Review by ZeroPointMax — APPROVED** [review]
👍  Danke :)

**Pull request merged**